contrib/qubes/doc/heads_udpate.md

```diff
@@ -0,0 +1,86 @@
+# Heads update
+
+The Heads update was tested on the `Lenovo ThinkPad x230`.
+
+## Requirements
+
+You need to build and flash Heads ROM from the
+[3mdeb fork](https://github.com/3mdeb/heads/tree/qubes-fwupd).
+Currently, you need to pass custom metadata URL:
+
+```
+https://fwupd.org/downloads/firmware-3c81bfdc9db5c8a42c09d38091944bc1a05b27b0.xml.gz
```

Comment:
is this a leaked private URL from LVFS?   It seems like it.  I don't think we want that in documentation.

Reply:
Deleted. It won't be needed when the Heads updates will appear in the LVFS production metadata.

Reply:
Perhaps it makes sense on the LVFS side to put them in "testing" channel for now.  And in the documentation you can refer to opting into the testing channel.  When it goes to stable you can change the documentation later on.